docs/api/web-contents.md

```diff
@@ -1182,6 +1182,8 @@ app.once('ready', () => {
 * `options` Object (optional)
   * `mode` String - Opens the devtools with specified dock state, can be
   `right`, `bottom`, `undocked`, `detach`. Defaults to last used dock state.
+  * `activate` Boolean (optional) - Whether to bring the opened devtools window to the
+    foreground. The default is `true`.
   In `undocked` mode it's possible to dock back. In `detach` mode it's not.
```

Comment:
This line should be right next to the `mode` option description.
Please add a description of the `activate` option below it.